### DISCUSSION TITLE
聊聊动漫《Girls Band Cry》 - 没有秃头基因的燊的博客

### DISCUSSION THREAD

**Discussion by giscus** [yzs020220/giscus-repo#25] [Announcements]
# 聊聊动漫《Girls Band Cry》 - 没有秃头基因的燊的博客

一开始只是通过看网上的视频之类的知道些梗，但最终还是决定去看了，确实很好看 - 没有秃头基因的燊 - 没有秃头基因的燊的博客

https://yzs020220.github.io/posts/34025/

**Comment by HCY-ASLEEP** [comment]
致消逝的回忆:)RIP

**Comment by STEVE-1936550490** [comment]
生活面前，梦想是反而不值一提的沉重
